int32 command_id
int32 send_ack
int32 lego_class
float64 coord_x
float64 coord_y
float64 coord_z
float64 rot_roll
float64 rot_pitch
float64 rot_yaw
